independent-publisher-connectors/MyHospitals by AIHW/apiProperties.json

```diff
@@ -0,0 +1,65 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#007ee5",

```

Comment:
@paulculmsee Please go thru the [template provided for apiProperties.json file for IP connectors](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json). The iconBrandColor must be "#da3b01".